conanfile.py

```diff
@@ -52,6 +50,11 @@ def config_options(self):
         elif not self.options.with_crash_handling:
             del self.options.crashdump_server
 
+    def build_requirements(self):
+        self.build_requires('protoc_installer/3.9.1@bincrafters/stable#0')
+        self.build_requires('grpc_codegen/1.27.3@orbitdeps/stable#ec39b3cf6031361be942257523c1839a')
+        self.build_requires('gtest/1.10.0#ef88ba8e54f5ffad7d706062d0731a40', force_host_context=True)
```

Comment:
what is `force_host_context`?

Reply:
Conan has two contexts which are often the same, but differ for cross compilation. Build requirements belong by default to the build context. In a cross compilation environment, the host context refers to the target, so it's Linux for the ggp_ configurations, while the build context is the platform you're compiling on (Windows for example). Since gtest needs to be linked in the host context, the package also needs to be from the host context (Linux library in that example.)

protoc_installer and grpc_codegen are code generator which have to run on the build machine, so they don't get the flag. We need Windows executables in that example.

Reply:
Makes a lot of sense. I was guessing it has to do with the crosscompilation